newIDE/app/src/UI/Dialog/index.js

```diff
@@ -7,6 +7,7 @@ const styles = {
   },
   noMarginBody: {
     padding: 0,
+    paddingBottom: 25, //required for tooltips to work
```

Comment:
Are you sure about this? Can you inspect dialogs using it (like BuildsDialog, ExportDialog, AboutDialog) to be sure we're not breaking the rendering of some?

Reply:
without it tooltips over bottom buttons were creating a scrollbar and getting cut off. I couldnt find a way to force the tooltips to appear above the cursor instead of below it

Reply:
it doesnt seem to create new problems elsewhere, but does expand the dialog size vertically a bit

Reply:
Can we remove the tooltip and remove this padding bottom?
Expanding the dialog size a bit seems a bad idea in terms of UI, and I think we can remove tooltips on buttons on the bottom of dialogs (they are plain text buttons, so should be understandable without tooltips).

Reply:
done :)